tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -257,6 +257,20 @@ public function bar($foo)
 #
 strlen($a);
 echo strlen($a);
+',
+            ],
+            'fix multiple calls in single code' => [
+                '<?php
+
+\json_encode($foo);
+\strlen($foo);
+\strlen($foo);
+',
+                '<?php
```

Comment:
This, unchanged `Tokens` collection is size of 20.
The fixer has to add 3 new tokens to this collection.

original code has to run `offsetSet` 45 times, as it has to move all tokens from strlen, then middle strlen, then from json_encode - which means that last strlen was moved 3 times!
my proposal has to run `offsetSet` only 20 times, as it has to move one token no more than once